### PR TITLE
Fix android tests.

### PR DESCRIPTION
**Pull request by inferno-chromium**

### CODE REVIEW

File: src/python/tests/core/platforms/android/device_test.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+from platforms.android import adb
+from platforms.android import device
+from system import environment
+from tests.test_libs import helpers as test_helpers
+from tests.test_libs import test_utils
+
+
+@test_utils.android_device_required
+class BaseDeviceTest(unittest.TestCase):
```
Comment (mbarbella-chromium) [inline src/python/tests/core/platforms/android/device_test.py]: Could we move this class to some place in test_libs?

Comment (inferno-chromium) [reply src/python/tests/core/platforms/android/device_test.py]: created android_helpers.py similar to untrusted_runner_helpers.py in test_libs.

**Pull request merged**